# Collision Status and Statistics Message
# if `is_collision` is True, then `object_name` will represent the object
#   with which the agent is colliding.

Header header

bool is_collision
string object_name
